### ISSUE TITLE
2025-01-28, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#65]
# Today's Arxiv Papers

(1) [Evaluating The Performance of Using Large Language Models to Automate Summarization of CT Simulation Orders in Radiation Oncology](http://arxiv.org/abs/2501.16309v1), 2025-01-27

(2) [RAPID: Retrieval-Augmented Parallel Inference Drafting for Text-Based Video Event Retrieval](http://arxiv.org/abs/2501.16303v1), 2025-01-27

(3) [Matryoshka Re-Ranker: A Flexible Re-Ranking Architecture With Configurable Depth and Width](http://arxiv.org/abs/2501.16302v1), 2025-01-27

(4) [Large Models in Dialogue for Active Perception and Anomaly Detection](http://arxiv.org/abs/2501.16300v1), 2025-01-27

(5) [FALCON: Resolving Visual Redundancy and Fragmentation in High-resolution Multimodal Large Language Models via Visual Registers](http://arxiv.org/abs/2501.16297v1), 2025-01-27

(6) [Brain-Adapter: Enhancing Neurological Disorder Analysis with Adapter-Tuning Multimodal Large Language Models](http://arxiv.org/abs/2501.16282v1), 2025-01-27

(7) [Do LLMs Have Visualization Literacy? An Evaluation on Modified Visualizations to Test Generalization in Data Interpretation](http://arxiv.org/abs/2501.16277v1), 2025-01-27

(8) [URAG: Implementing a Unified Hybrid RAG for Precise Answers in University Admission Chatbots -- A Case Study at HCMUT](http://arxiv.org/abs/2501.16276v1), 2025-01-27

(9) [Return of the Encoder: Maximizing Parameter Efficiency for SLMs](http://arxiv.org/abs/2501.16273v1), 2025-01-27

(10) [A foundation model for human-AI collaboration in medical literature mining](http://arxiv.org/abs/2501.16255v1), 2025-01-27

(11) [Multi-Agent Geospatial Copilots for Remote Sensing Workflows](http://arxiv.org/abs/2501.16254v1), 2025-01-27

(12) [Zero-Shot Decision Tree Construction via Large Language Models](http://arxiv.org/abs/2501.16247v1), 2025-01-27

(13) [Phase Transitions in Large Language Models and the $O(N)$ Model](http://arxiv.org/abs/2501.16241v1), 2025-01-27

(14) [AiGet: Transforming Everyday Moments into Hidden Knowledge Discovery with AI Assistance on Smart Glasses](http://dx.doi.org/10.1145/3706598.3713953), 2025-01-27

(15) [Language-Based Bayesian Optimization Research Assistant (BORA)](http://arxiv.org/abs/2501.16224v1), 2025-01-27

(16) [DBRouting: Routing End User Queries to Databases for Answerability](http://arxiv.org/abs/2501.16220v1), 2025-01-27

(17) [Enhancing Visual Inspection Capability of Multi-Modal Large Language Models on Medical Time Series with Supportive Conformalized and Interpretable Small Specialized Models](http://arxiv.org/abs/2501.16215v1), 2025-01-27

(18) [Provence: efficient and robust context pruning for retrieval-augmented generation](http://arxiv.org/abs/2501.16214v1), 2025-01-27

(19) [From Informal to Formal -- Incorporating and Evaluating LLMs on Natural Language Requirements to Verifiable Formal Proofs](http://arxiv.org/abs/2501.16207v1), 2025-01-27

(20) [Raiders of the Lost Dependency: Fixing Dependency Conflicts in Python using LLMs](http://arxiv.org/abs/2501.16191v1), 2025-01-27

(21) [SWIFT: Mapping Sub-series with Wavelet Decomposition Improves Time Series Forecasting](http://arxiv.org/abs/2501.16178v1), 2025-01-27

(22) [Will Systems of LLM Agents Cooperate: An Investigation into a Social Dilemma](http://arxiv.org/abs/2501.16173v1), 2025-01-27

(23) [MetaDecorator: Generating Immersive Virtual Tours through Multimodality](http://arxiv.org/abs/2501.16164v1), 2025-01-27

(24) [CITYWALK: Enhancing LLM-Based C++ Unit Test Generation via Project-Dependency Awareness and Language-Specific Knowledge](http://arxiv.org/abs/2501.16155v1), 2025-01-27

(25) [AdaCoT: Rethinking Cross-Lingual Factual Reasoning through Adaptive Chain-of-Thought](http://arxiv.org/abs/2501.16154v1), 2025-01-27

(26) [AI Agents for Computer Use: A Review of Instruction-based Computer Control, GUI Automation, and Operator Assistants](http://arxiv.org/abs/2501.16150v1), 2025-01-27

(27) [PATCH: Empowering Large Language Model with Programmer-Intent Guidance and Collaborative-Behavior Simulation for Automatic Bug Fixing](http://arxiv.org/abs/2501.16149v1), 2025-01-27

(28) [SampleLLM: Optimizing Tabular Data Synthesis in Recommendations](http://arxiv.org/abs/2501.16125v1), 2025-01-27

(29) [The Shiny Scary Future of Automated Research Synthesis in HCI](http://arxiv.org/abs/2501.16084v1), 2025-01-27

(30) [Integration of LLM Quality Assurance into an NLG System](http://arxiv.org/abs/2501.16078v1), 2025-01-27

(31) [PISCO: Pretty Simple Compression for Retrieval-Augmented Generation](http://arxiv.org/abs/2501.16075v1), 2025-01-27

(32) [Skeleton-Guided-Translation: A Benchmarking Framework for Code Repository Translation with Fine-Grained Quality Evaluation](http://arxiv.org/abs/2501.16050v1), 2025-01-27

(33) [PRISMe: A Novel LLM-Powered Tool for Interactive Privacy Policy Assessment](http://arxiv.org/abs/2501.16033v1), 2025-01-27

(34) [FDLLM: A Text Fingerprint Detection Method for LLMs in Multi-Language, Multi-Domain Black-Box Environments](http://arxiv.org/abs/2501.16029v1), 2025-01-27

(35) [TOPLOC: A Locality Sensitive Hashing Scheme for Trustless Verifiable Inference](http://arxiv.org/abs/2501.16007v1), 2025-01-27

(36) [Understanding Long Videos via LLM-Powered Entity Relation Graphs](http://arxiv.org/abs/2501.15953v1), 2025-01-27

(37) [TimeHF: Billion-Scale Time Series Models Guided by Human Feedback](http://arxiv.org/abs/2501.15942v1), 2025-01-27

(38) [SkillScope: A Tool to Predict Fine-Grained Skills Needed to Solve Issues on GitHub](http://arxiv.org/abs/2501.15922v1), 2025-01-27

(39) [Parametric Retrieval Augmented Generation](http://arxiv.org/abs/2501.15915v1), 2025-01-27

(40) [Robust Mobile Robot Path Planning via LLM-Based Dynamic Waypoint Generation](http://arxiv.org/abs/2501.15901v1), 2025-01-27

(41) [LCTG Bench: LLM Controlled Text Generation Benchmark](http://arxiv.org/abs/2501.15875v1), 2025-01-27

(42) [LLM-attacker: Enhancing Closed-loop Adversarial Scenario Generation for Autonomous Driving with Large Language Models](http://arxiv.org/abs/2501.15850v1), 2025-01-27

(43) [Aging-aware CPU Core Management for Embodied Carbon Amortization in Cloud LLM Inference](http://arxiv.org/abs/2501.15829v1), 2025-01-27

(44) [MADP: Multi-Agent Deductive Planning for Enhanced Cognitive-Behavioral Mental Health Question Answer](http://arxiv.org/abs/2501.15826v1), 2025-01-27

(45) [LemmaHead: RAG Assisted Proof Generation Using Large Language Models](http://arxiv.org/abs/2501.15797v1), 2025-01-27

(46) [Can Multimodal Large Language Models be Guided to Improve Industrial Anomaly Detection?](http://arxiv.org/abs/2501.15795v1), 2025-01-27

(47) [Harnessing Diverse Perspectives: A Multi-Agent Framework for Enhanced Error Detection in Knowledge Graphs](http://arxiv.org/abs/2501.15791v1), 2025-01-27

(48) [Large Language Models to Diffusion Finetuning](http://arxiv.org/abs/2501.15781v1), 2025-01-27

(49) [Is It Navajo? Accurate Language Detection in Endangered Athabaskan Languages](http://arxiv.org/abs/2501.15773v1), 2025-01-27

(50) [GraphICL: Unlocking Graph Learning Potential in LLMs through Structured Prompt Design](http://arxiv.org/abs/2501.15755v1), 2025-01-27

(51) [LLM-powered Multi-agent Framework for Goal-oriented Learning in Intelligent Tutoring System](http://arxiv.org/abs/2501.15749v1), 2025-01-27

(52) [IndicMMLU-Pro: Benchmarking the Indic Large Language Models](http://arxiv.org/abs/2501.15747v1), 2025-01-27

(53) [Gensors: Authoring Personalized Visual Sensors with Multimodal Foundation Models and Reasoning](http://dx.doi.org/10.1145/3708359.3712085), 2025-01-27

(54) [Adapting Biomedical Abstracts into Plain language using Large Language Models](http://arxiv.org/abs/2501.15700v1), 2025-01-26

(55) [TensorLLM: Tensorising Multi-Head Attention for Enhanced Reasoning and Compression in LLMs](http://arxiv.org/abs/2501.15674v1), 2025-01-26

(56) [People who frequently use ChatGPT for writing tasks are accurate and robust detectors of AI-generated text](http://arxiv.org/abs/2501.15654v1), 2025-01-26

(57) [Bringing Characters to New Stories: Training-Free Theme-Specific Image Generation via Dynamic Visual Prompting](http://arxiv.org/abs/2501.15641v1), 2025-01-26

(58) [HardML: A Benchmark For Evaluating Data Science And Machine Learning knowledge and reasoning in AI](http://arxiv.org/abs/2501.15627v1), 2025-01-26

(59) [Improving Estonian Text Simplification through Pretrained Language Models and Custom Datasets](http://arxiv.org/abs/2501.15624v1), 2025-01-26

(60) [Rethinking External Slow-Thinking: From Snowball Errors to Probability of Correct Reasoning](http://arxiv.org/abs/2501.15602v1), 2025-01-26

(61) [Evaluating an LLM-Powered Chatbot for Cognitive Restructuring: Insights from Mental Health Professionals](http://arxiv.org/abs/2501.15599v1), 2025-01-26

(62) [SedarEval: Automated Evaluation using Self-Adaptive Rubrics](http://arxiv.org/abs/2501.15595v1), 2025-01-26

(63) [SCP-116K: A High-Quality Problem-Solution Dataset and a Generalized Pipeline for Automated Extraction in the Higher Education Science Domain](http://arxiv.org/abs/2501.15587v1), 2025-01-26

(64) [Error Classification of Large Language Models on Math Word Problems: A Dynamically Adaptive Framework](http://arxiv.org/abs/2501.15581v1), 2025-01-26

(65) [Instruction Tuning for Story Understanding and Generation with Weak Supervision](http://arxiv.org/abs/2501.15574v1), 2025-01-26

(66) [Cross-Cultural Fashion Design via Interactive Large Language Models and Diffusion Models](http://arxiv.org/abs/2501.15571v1), 2025-01-26

(67) [ARWKV: Pretrain is not what we need, an RNN-Attention-Based Language Model Born from Transformer](http://arxiv.org/abs/2501.15570v1), 2025-01-26

(68) [Ocean-OCR: Towards General OCR Application via a Vision-Language Model](http://arxiv.org/abs/2501.15558v1), 2025-01-26

(69) [Commute Your Domains: Trajectory Optimality Criterion for Multi-Domain Learning](http://arxiv.org/abs/2501.15556v1), 2025-01-26

(70) [Advancing Generative Artificial Intelligence and Large Language Models for Demand Side Management with Electric Vehicles](http://arxiv.org/abs/2501.15544v1), 2025-01-26

(71) [Unveiling the Potential of Multimodal Retrieval Augmented Generation with Planning](http://arxiv.org/abs/2501.15470v1), 2025-01-26

(72) [Mind the Value-Action Gap: Do LLMs Act in Alignment with Their Values?](http://arxiv.org/abs/2501.15463v1), 2025-01-26

(73) [Data-adaptive Safety Rules for Training Reward Models](http://arxiv.org/abs/2501.15453v1), 2025-01-26

(74) [STATE ToxiCN: A Benchmark for Span-level Target-Aware Toxicity Extraction in Chinese Hate Speech Detection](http://arxiv.org/abs/2501.15451v1), 2025-01-26

(75) [OpenCharacter: Training Customizable Role-Playing LLMs with Large-Scale Synthetic Personas](http://arxiv.org/abs/2501.15427v1), 2025-01-26

(76) [The Potential of Large Language Models in Supply Chain Management: Advancing Decision-Making, Efficiency, and Innovation](http://arxiv.org/abs/2501.15411v1), 2025-01-26

(77) [Semantic Layered Embedding Diffusion in Large Language Models for Multi-Contextual Consistency](http://arxiv.org/abs/2501.15405v1), 2025-01-26

(78) [How Green are Neural Language Models? Analyzing Energy Consumption in Text Summarization Fine-tuning](http://arxiv.org/abs/2501.15398v1), 2025-01-26

(79) [Zero-Shot Interactive Text-to-Image Retrieval via Diffusion-Augmented Representations](http://arxiv.org/abs/2501.15379v1), 2025-01-26

(80) [How to Mitigate Information Loss in Knowledge Graphs for GraphRAG: Leveraging Triple Context Restoration and Query-Driven Feedback](http://arxiv.org/abs/2501.15378v1), 2025-01-26

(81) [Evaluating the Effectiveness of XAI Techniques for Encoder-Based Language Models](http://dx.doi.org/10.1016/j.knosys.2025.113042), 2025-01-26

(82) [Scaling Large Vision-Language Models for Enhanced Multimodal Comprehension In Biomedical Image Analysis](http://arxiv.org/abs/2501.15370v1), 2025-01-26

(83) [Decentralized Low-Rank Fine-Tuning of Large Language Models](http://arxiv.org/abs/2501.15361v1), 2025-01-26

(84) [Large Language Models as Theory of Mind Aware Generative Agents with Counterfactual Reflection](http://arxiv.org/abs/2501.15355v1), 2025-01-26
